.circleci/workflows.yml

```diff
@@ -695,6 +695,20 @@ jobs:
           condition: *deploy
           steps:
             - *skip
+  artifact-deployment:
+    docker:
+      - image: cimg/base:2024.01
+    steps:
+      - run:
+          name: Trigger bqetl-artifact-deployment in Airflow
+          command: >
+            curl --location "https://workflow.telemetry.mozilla.org/api/v1/dags/bqetl_artifact_deployment/dagRuns"
+            --header "Content-Type: application/json"
+            --header "Accept: application/json"
+            --header "Authorization: Basic ${AIRFLOW_BQETL_CI_USER_BASIC_AUTH}"
+            --data "{
+              \"note\": \"DAG Triggered by **[${CIRCLE_USERNAME}](https://github.com/${CIRCLE_USERNAME})** from ${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} CI build [${CIRCLE_BUILD_NUM}](${CIRCLE_BUILD_URL}).\"
+            }"
```

Comment:
Thought for the future: it'd be nice to have this be more targeted and deploy (only) the required assets. This would require DAG changes as well as changes here though

Reply:
Yeah, there is definitely more complexity to figure out what has actually changed and needs to be deployed